README.md

```diff
@@ -47,15 +47,15 @@ Supports only Lovelace UI. Last tested in 0.95.4.
 1. Download this repository as a ZIP (green button, top right) and unzip the archive
 2. Copy `/custom_components/mail_and_packages` to your `<config_dir>/custom_components/` directory
    * You will need to create the `custom_components` folder if it does not exist
-   * On Hassio the final location will be `/config/custom_components/mail_and_packages`
+   * On HomeAssistant (formerly hass.io) the final location will be `/config/custom_components/mail_and_packages`
    * On Hassbian the final location will be `/home/homeassistant/.homeassistant/custom_components/mail_and_packages`
-3. Copy/move `image-no-mailpieces700.jpg` and `mail_none.gif` to the www directory (recommend /www/mail_and_packages)
+3. Copy/move `image-no-mailpieces700.jpg` and `mail_none.gif` to the www directory (recommend `/www/mail_and_packages` as the path)
```

Comment:
Can we put this as recommend `<config_dir>/www/mail_and_packages`?

Reply:
Yeah makes sense.